src/proto/Polkadot.proto

```diff
@@ -142,7 +142,7 @@ message SigningInput {
     bytes private_key = 8;
 
     // Network type
-    Network network = 9;
+    uint32 network = 9;
```

Comment:
this is a breaking change for end user app, but required for future runtime support

Reply:
For breaking changes, need to mention it in next release notes (how to use new code)